openslides/utils/projector.py

```diff
@@ -104,3 +80,23 @@ def get_collection_elements_required_for_this(self, collection_element, config_e
             else:
                 output = []
         return output
+
+
+projector_elements = []  # type: List[ProjectorElement]
```

Comment:
The type could be Dict[str, ProjectorElement] where the key is the name of the projector element. Currently we call get_all_projector_elements() three times and in any of this cases we create this dict.

What do you think?

Reply:
Ok.